src/expr.rs

```diff
@@ -320,6 +320,19 @@ fn rewrite_closure(capture: ast::CaptureClause,
     Some(format!("{} {}", prefix, try_opt!(body_rewrite)))
 }
 
+fn nop_block_collapse(bl_str_o: Option<String>) -> Option<String> {
+    if let Some(ref bl_str) = bl_str_o {
+        if bl_str.starts_with("{") { //necessary to catch \"a => (), \" i.e. return empty tuple
+            let close_idx = bl_str.len() - 1;
+            let fnws_idx = bl_str[1..].find(|c: char| !c.is_whitespace()).unwrap() + 1;
+            if fnws_idx == close_idx {
+                return Some("{}".to_string());
+            }
+        }
+    }
+    return bl_str_o;
+}
+
```

Comment:
I would prefer longer names, e.g., `block_str` instead of `bl_str`, and I think you should cut the `_o` suffix, it's common Rust style to reuse variable names.